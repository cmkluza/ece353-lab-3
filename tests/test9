Tests read-after-write (RAW) data hazards without errors, with m = n = c = 2
./sim-mips -b 2 2 2 ./tests/asm/test9.s ./tests/out/test9.out
./tests/sols/test9.sol