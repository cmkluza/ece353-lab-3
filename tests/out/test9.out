program name: ./tests/asm/test9.s
stage utilization: 0.486486  0.216216  0.432432  0.000000  0.216216 
register values 0  0  0  0  0  0  0  15  25  20  45  70  1050  980  70  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  0  36
